interactors/checkbox.js

```diff
@@ -0,0 +1,38 @@
+import { CheckBox, createInteractor, perform } from '@bigtest/interactor';
+
+// The built-in CheckBox interactor expects an input element,
+// and anything added in the future would likely not work without modification.
+// Explicitly import known working functions.
+const { title, visible, focused } = CheckBox().specification.filters;
```

Comment:
Does using this checkbox filter directly work? I would think that we'd want to see if the FOLIO checkbox component _contained_ the focus element:

```js
focus: el => el.contains(el.ownerDocument.activeElement)
```

Reply:
Good point, that is likely more all encompassing.